### DISCUSSION TITLE
failed to import

### DISCUSSION THREAD

**Discussion by JerichoKuskanto** [jsvine/pdfplumber#1125] [Q&A]
so i installed pdfplumber using pip install pdfplumber in my CMD, when i try to import it into my .py it VScode give a warn import "pdfplumber" could not be resolved. i thought it just usual still gonna work warn. When i run my program it gives me ModuleNotFoundError: No module named 'pdfplumber'. i tried reinstalling using --user and the result still the same

**Comment by jsvine** [comment]
My hunch is that this is not an issue specific to `pdfplumber`, given the description. One way to check this: Do you encounter similar problems with other programs you haven't installed previously?

**Reply by JerichoKuskanto** [reply]
no, i have never encountered this kind of problem before

**Comment by salted-butter-joshua** [comment]
ImportError: DLL load failed while importing _rust: 找不到指定的程序。

**Reply by salted-butter-joshua** [reply]
cryptography的版本不对导致的
安装低版本的 cryptography
pip install cryptography==41.0.2
